src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import core.basesyntax.db.FruitTransaction;
```

Comment:
The import statement for 'FruitTransaction' seems unnecessary as this class is not used directly in the Main class.